### PR TITLE
Added metadata arg

### PR DESCRIPTION
**Pull request by kompotkot**
Part of PR https://github.com/bugout-dev/locust/pull/34

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -18,8 +18,9 @@ then
 fi
 
 REPO_URL=$(locust.github repo)
+COMMENTS_URL=$(cat $GITHUB_EVENT_PATH | jq -r .pull_request._links.comments.href)
 
-locust --format "$FORMAT" -r "$REPO" "$INITIAL_REF" "$TERMINAL_REF" --github "${REPO_URL}" | tee /locust.summary
+locust --format "$FORMAT" -r "$REPO" "$INITIAL_REF" "$TERMINAL_REF" --github "${REPO_URL}" --metadata "{\"comments_url\": \"${COMMENTS_URL}\", \"terminal_hash\": \"$TERMINAL_REF\"}" | tee /locust.summary
```
Comment (zomglings) [inline run.sh:23]: @kompotkot : I suggest to add an option or environment variable (e.g. `BUGOUT_SEND_SUMMARY=true`) in `run.sh`. Default should be `false` and if value is `false`, we shouldn't add metadata and we shouldn't send any information to Bugout API.

If users have set it to `true`, then we do all this additional stuff.

Comment (zomglings) [reply run.sh:23]: <html><body><h2><a href="https://github.com/simiotics/locust">Locust</a> summary</h2><h3>Git references</h3><b>Initial: </b><span>a2aa908</span><br><b>Terminal: </b><span>52a61c3</span><br><hr><div><details><summary><a href="locust/parse.py">locust/parse.py</a></summary><div><b>Changes:</b><ul><li><b>Name: </b><a href="locust/parse.py">LocustVisitor</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>16</span><br><b>Total lines: </b><span>58</span><br><b>Changes:</b><ul><li><b>Name: </b><a href="locust/parse.py">LocustVisitor.__init__</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>1</span><br><b>Total lines: </b><span>3</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">LocustVisitor._visit_class_or_function_def</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>7</span><br><b>Total lines: </b><span>27</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">LocustVisitor.patch_definitions</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>7</span><br><b>Total lines: </b><span>8</span><ul></ul></li></ul></li><li><b>Name: </b><a href="locust/parse.py">RawDefinition</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>1</span><br><b>Total lines: </b><span>8</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">LocustChange</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>1</span><br><b>Total lines: </b><span>9</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">RunResponse</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>6</span><br><b>Total lines: </b><span>6</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">definitions_by_patch</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>12</span><br><b>Total lines: </b><span>12</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">locust_changes_in_patch</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>53</span><br><b>Total lines: </b><span>57</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">calculate_changes</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>11</span><br><b>Total lines: </b><span>11</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">calculate_python_changes</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>3</span><br><b>Total lines: </b><span>3</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">calculate_changes_from_file</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>13</span><br><b>Total lines: </b><span>13</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">calculate_plugin_changes</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>36</span><br><b>Total lines: </b><span>38</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">run</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>12</span><br><b>Total lines: </b><span>12</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">populate_argument_parser</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>12</span><br><b>Total lines: </b><span>12</span><ul></ul></li><li><b>Name: </b><a href="locust/parse.py">main</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>31</span><br><b>Total lines: </b><span>31</span><ul></ul></li></ul></div></details></div><div><details><summary><a href="locust/render.py">locust/render.py</a></summary><div><b>Changes:</b><ul><li><b>Name: </b><a href="locust/render.py">generate_render_html</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>3</span><br><b>Total lines: </b><span>30</span><br><b>Changes:</b><ul><li><b>Name: </b><a href="locust/render.py">generate_render_html.render_html</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>3</span><br><b>Total lines: </b><span>25</span><ul></ul></li></ul></li><li><b>Name: </b><a href="locust/render.py">enrich_with_github_links</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>10</span><br><b>Total lines: </b><span>29</span><br><b>Changes:</b><ul><li><b>Name: </b><a href="locust/render.py">enrich_with_github_links._enrich_changes</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>8</span><br><b>Total lines: </b><span>8</span><ul></ul></li></ul></li><li><b>Name: </b><a href="locust/render.py">NestedChange</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>2</span><br><b>Total lines: </b><span>4</span><ul></ul></li><li><b>Name: </b><a href="locust/render.py">enrich_with_metadata</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>7</span><br><b>Total lines: </b><span>7</span><ul></ul></li><li><b>Name: </b><a href="locust/render.py">populate_argument_parser</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>29</span><br><b>Total lines: </b><span>29</span><ul></ul></li><li><b>Name: </b><a href="locust/render.py">run</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>21</span><br><b>Total lines: </b><span>21</span><ul></ul></li><li><b>Name: </b><a href="locust/render.py">main</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>31</span><br><b>Total lines: </b><span>31</span><ul></ul></li></ul></div></details></div><div><details><summary><a href="tests/test_git.py">tests/test_git.py</a></summary><div><b>Changes:</b><ul><li><b>Name: </b><a href="tests/test_git.py">TestLocustGit</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>21</span><br><b>Total lines: </b><span>21</span><br><b>Changes:</b><ul><li><b>Name: </b><a href="tests/test_git.py">TestLocustGit.test_git_run</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>18</span><br><b>Total lines: </b><span>18</span><ul></ul></li></ul></li></ul></div></details></div><div><details><summary><a href="tests/test_parse.py">tests/test_parse.py</a></summary><div><b>Changes:</b><ul><li><b>Name: </b><a href="tests/test_parse.py">TestLocustParse</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>29</span><br><b>Total lines: </b><span>29</span><br><b>Changes:</b><ul><li><b>Name: </b><a href="tests/test_parse.py">TestLocustParse.test_parse_run</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>26</span><br><b>Total lines: </b><span>26</span><ul></ul></li></ul></li></ul></div></details></div><div><details><summary><a href="tests/test_render.py">tests/test_render.py</a></summary><div><b>Changes:</b><ul><li><b>Name: </b><a href="tests/test_render.py">TestLocustRender</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>23</span><br><b>Total lines: </b><span>23</span><br><b>Changes:</b><ul><li><b>Name: </b><a href="tests/test_render.py">TestLocustRender.test_render_run</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>22</span><br><b>Total lines: </b><span>22</span><ul></ul></li></ul></li></ul></div></details></div><div><details><summary><a href="locust/ci_helpers/github.py">locust/ci_helpers/github.py</a></summary><div><b>Changes:</b><ul><li><b>Name: </b><a href="locust/ci_helpers/github.py">main</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>1</span><br><b>Total lines: </b><span>30</span><ul></ul></li></ul></div></details></div><div><details><summary><a href="locust/cli.py">locust/cli.py</a></summary><div><b>Changes:</b><ul><li><b>Name: </b><a href="locust/cli.py">generate_argument_parser</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>4</span><br><b>Total lines: </b><span>17</span><ul></ul></li><li><b>Name: </b><a href="locust/cli.py">main</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>8</span><br><b>Total lines: </b><span>15</span><ul></ul></li></ul></div></details></div><div><details><summary><a href="locust/git.py">locust/git.py</a></summary><div><b>Changes:</b><ul><li><b>Name: </b><a href="locust/git.py">LineInfo</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>1</span><br><b>Total lines: </b><span>5</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">HunkBoundary</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>4</span><br><b>Total lines: </b><span>4</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">HunkInfo</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>4</span><br><b>Total lines: </b><span>6</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">PatchInfo</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>3</span><br><b>Total lines: </b><span>6</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">RunResponse</a><br><b>Type: </b><span>class</span><br><b>Changed lines: </b><span>5</span><br><b>Total lines: </b><span>5</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">get_patches</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>12</span><br><b>Total lines: </b><span>30</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">hunk_boundary</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>26</span><br><b>Total lines: </b><span>26</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">process_hunk</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>6</span><br><b>Total lines: </b><span>22</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">revision_file</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>7</span><br><b>Total lines: </b><span>38</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">populate_argument_parser</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>21</span><br><b>Total lines: </b><span>21</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">run</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>19</span><br><b>Total lines: </b><span>19</span><ul></ul></li><li><b>Name: </b><a href="locust/git.py">main</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>24</span><br><b>Total lines: </b><span>24</span><ul></ul></li></ul></div></details></div><div><details><summary><a href="locust/version.py">locust/version.py</a></summary><div><b>Changes:</b><ul><li><b>Name: </b><a href="locust/version.py">main</a><br><b>Type: </b><span>function</span><br><b>Changed lines: </b><span>1</span><br><b>Total lines: </b><span>2</span><ul></ul></li></ul></div></details></div></body></html>

File: run.sh [hunk 2]
```diff
@@ -18,8 +18,9 @@ then
 fi
 
 REPO_URL=$(locust.github repo)
+COMMENTS_URL=$(cat $GITHUB_EVENT_PATH | jq -r .pull_request._links.comments.href)
```
Comment (zomglings) [inline run.sh:21]: Remove `jq` dependency by using `python -c`.

**Comment by kompotkot** [comment]
PR re-created:
https://github.com/bugout-dev/locust-action/pull/7/files

File: run.sh (hunk 2 above)
Comment (kompotkot) [reply run.sh:21]: Done

**Pull request closed without merging**